ruby/example_code/iam/scenario_create_user_assume_role.rb

```diff
@@ -0,0 +1,295 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# Purpose
+#
+# Shows how to use AWS SDK for Ruby to create an IAM user, assume a role,
+# and perform AWS actions.
```

Comment:
Shows how to use the AWS SDK for Ruby to create an AWS Identity and Access Management (IAM) user, assume a role, and perform AWS actions.